### PR TITLE
ADLS g1 to g2 - demo migration steps

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

**Comment by benradstone** [comment]
Also included a change to the Troubleshooting link, as they were all going to the HDP Sandbox one instead of General.

File: docs/quickstarts/installation/adlsg1-adlsg2.md [hunk 1]
```diff
@@ -128,10 +128,43 @@ Log in to your VM prior to starting these steps.
 
 ## Migration
 
-You can now create a [replication rule](../operation/create-rule.md) and then [migrate your data](../operation/migration.md).
+You can now test migration of data between your ADLS Gen1 and Gen2 storage using Fusion's LiveMigrator.
```
Comment (catrionabarker) [inline docs/quickstarts/installation/adlsg1-adlsg2.md]: To keep it consistent with the other qs maybe:
Follow the steps below to demonstrate the migration of data from your ADLS Gen1 to Gen2 storage.

'between' to me suggests bidirectional which it isn't.

File: docs/quickstarts/installation/adlsg1-adlsg2.md [hunk 2]
```diff
@@ -128,10 +128,43 @@ Log in to your VM prior to starting these steps.
...
+
+### Prepare sample data
+
+[Upload](https://docs.microsoft.com/en-us/azure/data-lake-store/data-lake-store-get-started-portal#uploaddata) sample data to your ADLS Gen1 storage account. You can get the sample data from the link below:
```
Comment (catrionabarker) [inline docs/quickstarts/installation/adlsg1-adlsg2.md]: 'Upload' being a link to MS docs isn't visually very clear to me.

Maybe:
Upload sample data to your ADLS Gen1 storage account, see the Microsoft docs for details on how to do this.
You can get the sample data from:
customer_addresses_dim.tsv.gz

File: docs/quickstarts/installation/adlsg1-adlsg2.md [hunk 3]
```diff
@@ -128,10 +128,43 @@ Log in to your VM prior to starting these steps.
...
+* Rule Name = `migration`
+* Path for all zones = `/`
+* Default exclusions
+* Preserve HCFS Block Size = *False*
+
+### Migrate your data
+
+1. On the dashboard, view the `migration` rule.
+
+2. Start your migration with the following overwrite settings:
```
Comment (catrionabarker) [inline docs/quickstarts/installation/adlsg1-adlsg2.md]: just 'settings', not 'overwrite settings'

**Pull request merged**